### ISSUE TITLE
a11y bug in slider block - Button should not have `tabIndex=-1`

### ISSUE THREAD

**Issue by sneridagh** [kitconcept/volto-light-theme#248]
* as anonymous user
* go to Homepage
* -> actual behavior (You cannot navigate properly using keyboard in the slider and reach the button)
* -> behavior you would expect You can actually tab and reach the button

Slider in https://light-theme.kitconcept.io/

This was reported by David Ichim, a member of the Plone Community after watching the conference talk, and test driving the dlr.de website. He's a EEA team member, taking care of a11y issues in their sites.

**Comment by sneridagh** [comment]
Hint by David: 
Remove the -1 and use:

```
.slick-slide {
  visibility: hidden;
}
.slick-active {
  visibility: visible;
}
```

**Comment by ichim-david** [comment]
@sneridagh I think you should also remove it from the link, if the user hides the button the link also has tabindex=-1 so no keyboard-accessible way of interacting with the slider to go to the attached link.

**Comment by sneridagh** [comment]
Fixed in 3.0.0a0 by using the new slider add-on.

**Comment by davisagli** [comment]
@sneridagh Was it? On https://light-theme.kitconcept.io/ I can't figure out how to reach the "continue reading" button with the keyboard.
